packages/next-on-pages/docs/caching.md

```diff
@@ -2,6 +2,10 @@
 
 `@cloudflare/next-on-pages` comes with support for data revalidation and caching for fetch requests. This is done in our router and acts as an extension to Next.js' built-in functionality.
 
+> [!NOTE]
+> This cache is persisted across deployments inline with the standard behaviour. You are responsible for revalidating/purging this cache. It is not handled for you by `@cloudflare/next-on-pages` or Cloudflare Pages.
```

Comment:
```suggestion
> This cache is persisted across deployments inline with what the [Next.js documentation states](https://nextjs.org/docs/app/building-your-application/caching#data-cache). If you don't want such behaviour, you are responsible for revalidating/purging this cache. It is not handled for you by `@cloudflare/next-on-pages` or Cloudflare Pages.
```

Reply:
They are responsible for managing it even if they do want this though

> If you don't want such behaviour, you are responsible for revalidating/purging this cache.

Reply:
good point 🙂👍

Reply:
```suggestion
> This cache is persisted across deployments inline with what the [Next.js documentation states](https://nextjs.org/docs/app/building-your-application/caching#data-cache).
```

Reply:
I still think it's important to explicitly call out they are responsible for managing. What's your thinking to not include that part?

Reply:
Sorry, bad copy paste on my part! 😓 🙇 

(I meant to remove my latter suggested change and keep only the first part 😅)

```suggestion
> This cache is persisted across deployments inline with what the [Next.js documentation states](https://nextjs.org/docs/app/building-your-application/caching#data-cache). You are responsible for revalidating/purging this cache. It is not handled for you by `@cloudflare/next-on-pages` or Cloudflare Pages.
```